cmd/skopeo/proxy.go

```diff
@@ -701,6 +702,17 @@ func (h *proxyHandler) FinishPipe(args []interface{}) (replyBuf, error) {
 	return ret, err
 }
 
+// close releases all resources associated with this proxy backend
+func (h *proxyHandler) close() {
+	for _, image := range h.images {
+		err := image.src.Close()
+		if err != nil {
+			// This shouldn't be fatal
+			logrus.Warnf("failed to close image: %v", err)
```

Comment:
Non-blocking:
```suggestion
			logrus.Warnf("Failed to close image %s: %v", transports.ImageName(image.Reference()), err)
```
in case there are multiple images; but this should basically never happen anyway…

… and `CloseImage` is not reporting the error at all, so focusing on this case is not really warranted.